### DISCUSSION TITLE
New feed doesn't take into account how people actually use it

### DISCUSSION THREAD

**Discussion by mikesprague** [community/community#66166] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For example, I have thousands of stars bc I star repos as a form of bookmarking. I don't want to see every release from every repo I have starred.

The amount of noise you have introduced into the feeds makes them unusable.

I get that this is how you want devs to use the feed but many of us aren't new to GH and have lots of people we follow and repos we've starred or watched.

This change makes the GH homepage much less useful.

EDIT:

Looking at things closer, I don't currently see any star activity in my feed. Almost as if this change has been released with bugs/issues.

I am also confused by the algorithm and ordering of the feed. Much of it looks to be reverse chronological but then there's stuff from 3 weeks ago mixed in with stuff from 12 hours ago. In addition, it showed me a release I did a couple days ago in one project but isn't showing one from last week (in a more popular project) despite being "weeks" into the feed.

I'd like to also point out that many new features go to "feature preview" first but this one just "happened" per se. And even worse, it sure doesn't seem to be fully baked.

EDIT:

The new feed is also very slow. It often hasn't completed loading by the time I move on to another location within GH.

**Comment by mewmew** [comment]
I understand the wish of GH to experiment with the feed and add options to discover new repos and activity of old repos. However, in it's current state it's not possible to use without drowning in information.

I have exactly the same issue as outlined by @mikesprague. I currently have 2k+ stars (i.e. bookmarks), and as a result the feed is absolutely flooded with activity/releases by these stared repos.

I want to receive activity/release info about repositories I *watch* or activity of people I *follow*. Not about repos I star.

With the latest update, the feed went from semi-useful to useless unfortunately. At least for my use case (using stars as "bookmarks").

One solution may be to add a toggle to ignore stared repos.

With kindness,
Robin

**Comment by mikesprague** [comment]
Saw this on another post. Not sure how long it will remain alive and it looks a little weird full width but you can view previous version here for now: https://github.com/dashboard-feed

**Reply by mikesprague** [reply]
Comment added with some userstyles to tweak the width https://github.com/orgs/community/discussions/66166#discussioncomment-6937511

**Comment by shailshouryya** [comment]
I [made a post](https://github.com/orgs/community/discussions/66226) with links to current discussions about this (will continue to update over time)

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Comment by mewmew** [comment]
> Saw this on another post. Not sure how long it will remain alive and it looks a little weird full width but you can view previous version here for now: https://github.com/dashboard-feed

Thanks @mikesprague! Very useful. I hope the `dashboard-feed` feature is kept alive, as it contains the information I want and more importantly it does not contain the information I don't want in my feed.

Cheers,
Robin

**Comment by mikesprague** [comment]
For those viewing https://github.com/dashboard-feed directly... 

If you have a UserStyles manager (like [Stylus](https://github.com/openstyles/stylus/)) installed, you can create a userstyle with the following content to give that page a little breathing room...

EDIT: use @sinsinpub's solution (example below) from comment https://github.com/orgs/community/discussions/66166#discussioncomment-6939769 - my previous example was a bit over-engineered lol and more prone to break with markup changes

```css
.application-main {
  margin: 0 100px;
}
```
:point_up: _adjust the 100px to your liking_

Which should make https://github.com/dashboard-feed look something like this instead of full width (showing bottom of my feed bc of private repo content at the top but you get the idea 🙂):
![gh-feed-userstyles-example](https://github.com/community/community/assets/560705/7bb0842c-76c1-4dfd-9051-6ab44ba68c13)

**Reply by sinsinpub** [reply]
`.application-main { margin: 0 60px; }` one line is enough... (adjust to greater value if more space preferred)

**Comment by Nanguage** [comment]
New feed completely cuts off the connection between people, and for me, the significance of the GitHub homepage lies in being able to let me know what my followed users have starred and what they are interested in. New feed lacks this core functionality.

**Reply by mikesprague** [reply]
Agreed. Part of the community aspect for me was seeing what the devs I followed were interested in and discovering new projects that way. Along with missing commits and not seeing who's starred your own repos etc. they have fundamentally broken how I have used GitHub's homepage for years.

**Comment by mikesprague** [comment]
Couple times today (including currently) the new feed has been starting at content that's 2 months old for me. It seems like some sort of caching issue but who knows. Add it to the list of issues with this new feed... sigh.

**Comment by Hexstream** [comment]
Activity on starred but not watched repos is obviously 100% irrelevant.
How can GitHub possibly not understand this?

**Comment by simov** [comment]
I think I will be using the https://github.com/dashboard-feed as well. Another thing that is missing in the new feed is the activity in the organizations that I am member of. So besides the rest of the issues outlined above now I have to additionally go and click on my org feed just to not miss something there. Which I obviously forget to do and I do miss stuff.

**Reply by mikesprague** [reply]
Same here. Part of three orgs for work and having that content on the main page was incredibly useful. Based on zero comments from GH on all of this, seems these feed changes are here to stay 😕

**Comment by mikesprague** [comment]
Still a lot of content missing but all of the sudden today my repos being starred by others have started to appear in the new feed... I'll still be using https://github.com/dashboard-feed but it's a start

**Reply by mikesprague** [reply]
And they're gone 🙁...  must be they're actively tweaking things

**Reply by ghost** [reply]
its showing for me too. it started today or yesterday and hasn't gone away like yours

**Comment by mikesprague** [comment]
Just noticed this update to the feed showing people I follow starring stuff (but seems to be some weird caching issues as it's old data), perhaps a step in the right direction...

<img width="927" alt="Screenshot 2023-09-27 at 08 36 59" src="https://github.com/community/community/assets/560705/51100b6d-4d89-4635-bab8-4d0eabfc623d">

**Comment by tvrusso** [comment]
The new feed is absolutely worthless.  I'm getting information about every time someone I follow has starred anything, but nothing about new pushes to codes I'm watching or pushes from people I am watching.  

I use github to keep track of what is happening to projects I care about, not get random noise about what *other* people care about.  I could live with the noise about who starred what, but not at the expense of what is happening to projects I chose to watch.
